manim/utils/testing/frames_comparison.py

```diff
@@ -81,6 +83,9 @@ def decorator_maker(tested_scene_construct):
         @functools.wraps(tested_scene_construct)
         # The "request" parameter is meant to be used as a fixture by pytest. See below.
         def wrapper(*args, request: FixtureRequest, tmp_path, **kwargs):
+            # check for cairo version
+            if renderer_class is CairoRenderer and cairo.cairo_version() < 11800:
```

Comment:
The discrepancy between 11800 here and the `1.18.0` string in the build script is somewhat unfortunate -- but should this here perhaps be stored in a (constant) variable too?